charts/helm_lib/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
 type: library
 name: deckhouse_lib_helm
-version: 1.2.0
```

Comment:
Please rebase on main and increase version. We already have 1.3.0 version [here](https://github.com/deckhouse/lib-helm/pull/31/files#diff-f196d3c8984f4ac7d04a2db44baa7ca35f0a96d1c74d3ee9aad3a1a9ee0e6d0eL4)